setup.py

```diff
@@ -88,7 +88,7 @@ def run(self):
         "Topic :: Software Development :: Compilers",
         "Topic :: Software Development :: Libraries"
         ],
-    setup_requires=["Opcodes==0.3.9"],
+    setup_requires=["Opcodes==0.3.9", "six"],
```

Comment:
Good find! I put it in when before I understood that the peachpy import needed to be removed. Unfortunately, determining the version comes before setup_requires can be evaluated and install six, so it's not helpful. Removed.